plugins/crypto_airswap/crypto_airswap.go

```diff
@@ -0,0 +1,569 @@
+package main
+
+import (
+	"autonity-oracle/plugins/common"
+	"autonity-oracle/plugins/crypto_airswap/erc20"
+	swaperc20 "autonity-oracle/plugins/crypto_airswap/swap_erc20"
+	"autonity-oracle/types"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	ecommon "github.com/ethereum/go-ethereum/common"
+	types2 "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/event"
+	"github.com/hashicorp/go-hclog"
+	ring "github.com/zfjagann/golang-ring"
+	"math/big"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+)
+
+var (
+	orderBookCapacity = 64
+	version           = "v0.0.1"
+	ATNUSDC           = "ATN-USDC"
+	NTNUSDC           = "NTN-USDC"
+	supportedSymbols  = []string{ATNUSDC, NTNUSDC}
+	priceFile         = "airswap_prices.json"
+	NTNTokenAddress   = types.AutonityContractAddress // Autonity contract is the protocol contract of NTN token
+	// todo: set an initial price of NTN-USDC on genesis, there is no price at all on airswapERC20 on genesis until there
+	// is an exchange/swap happens. Otherwise, the oracle client might be slashed due to the missing of price of a symbol.
+	NTNPriceOnGenesis = "" // The NTN-USDC price on the genesis phase
```

Comment:
Okay, in this case, the oracle operator should enable at least uniswap plugin to source NTN-USDC and ATN-USDC at the genesis phase, otherwise without the uniswap plugin, the oracle might be slashed due to missing data point of NTN-USDC and ATN-USDC. I think we need to add this to the document. I will remove the dependency of the genesis price in dex-plugin.

Reply:
Resolved in ba239c11cecce8b9e0ee46f6b7dc1549a50b9879, we remove the dependency of genesis price in dex-plugin.